charts/stable/appdaemon/questions.yaml

```diff
@@ -73,6 +73,19 @@ questions:
                     - value: "OnDelete"
                       description: "(Legacy) OnDelete: ignore .spec.template changes"
 # Include{controllerExpert}
+  - variable: secret
+    group: "Container Configuration"
+    label: "Image Secrets"
+    schema:
+      type: dict
+      attrs:
+        - variable: TOKEN
+          label: "HA Token"
+          description: "Your HomeAssistant Token"
+          schema:
+            type: string
+            default: ""
+            required: true
```

Comment:
Things like these, if we fill this with a placeholder for now, that at least doesn't instantly break the update on update.
We can remove those on a later update anyway.

It's still somewhat breaking (as people have to edit and change it after update), but at least, in most cases, it should finish the update.